### PR TITLE
fix-menu

### PR DESCRIPTION
**Pull request by sutaungmai**

### CODE REVIEW

**Comment by sutaungmai** [comment]
public/index.html add font-size
src/ResponsiveDrawer.js

File: public/index.html [hunk 1]
```diff
@@ -31,6 +31,7 @@
     <style>
       * {
         margin: 0;
+        font-size: small;
```
Comment (zay467) [inline public/index.html:34]: Why is the font size set here? This is changing the font size of the whole app. It's might result in unwanted behavior or changes in the rest of the app.

File: src/components/ResponsiveDrawer.js [hunk 2]
```diff
@@ -47,18 +50,26 @@ const ResponsiveDrawer = ({
...
 
   const drawer = (
     <>
       <Toolbar />
       <Divider />
-      <List>
-        <ListItem onClick={openBillingAccordionList}>
-          <Button
-            size="small"
+      <List
```
Comment (zay467) [inline src/components/ResponsiveDrawer.js:61]: The whole design is nothing like Lab. 
image

You can reference the code from here. I also attached this link in trello.

https://github.com/thamardaw/magix-support-lab-management-web/blob/main/src/components/ResponsiveDrawer.js

**Review by zay467 — CHANGES_REQUESTED** [review]
Please read the task carefully.

https://trello.com/c/DMXR0jyg/20-dev-billing-fe-12520222026-update-billings-drawer-as-labs-drawer

**Review by zay467 — APPROVED** [review]

**Pull request merged**